include/ImageProcessing/Kernels.h

```diff
@@ -156,6 +160,22 @@ static float random15x15KernelAlign[225] = {0, 7, 6, 8, 5, 9, 6, 1, 6, 0, 7, 5,
                                             9, 4, 6, 5, 4, 3, 6, 7, 5, 4, 1, 0, 6, 0, 6,
                                             3, 1, 1, 2, 0, 8, 5, 7, 6, 1, 1, 0, 7, 9, 6,
                                             5, 0, 2, 1, 6, 8, 6, 4, 5, 3, 1, 9, 3, 7, 9};
+
+static uint8_t random15x15KernelAlignInt[225] = {0, 7, 6, 8, 5, 9, 6, 1, 6, 0, 7, 5, 6, 1, 6,
```

Comment:
What is the use of this kernel?

Reply:
I just thought of adding an extra kernel I'll remove it if you say so

Reply:
Please remove it as it is not different than the previous (already present) kernel.